src/calva-fmt/src/providers/ontype_formatter.ts

```diff
@@ -34,7 +34,13 @@ export class FormatOnTypeEditProvider implements vscode.OnTypeFormattingEditProv
     const editor = util.getActiveTextEditor();
 
     const pos = editor.selection.active;
-    if (vscode.workspace.getConfiguration('calva.fmt').get('formatAsYouType')) {
+    if (
+      vscode.workspace.getConfiguration('editor').get('formatOnType') ||
+      vscode.workspace
+        .getConfiguration('editor')
+        .inspect('formatOnType')
+        .languageIds.includes(document.languageId)
```

Comment:
Is this part of this setting? I only see it as a boolean in the docs.

Reply:
This is the built-in VS Code setting. It's a boolean, yes.

Reply:
I'm wondering how you can call `.languageIds.includes(document.languageId)` on a boolean value. 🤔

Reply:
Oh nevermind, I have educated myself with the docs. 😃

Reply:
Haha. Yes, it's a bit funny this API. I often use Joyride to educate myself about vscode:

<img width="1219" alt="image" src="https://user-images.githubusercontent.com/30010/202640846-984beef8-7dac-40e6-86de-bcd9cebf50b8.png">

Which is interesting, because I often use TypeScript to do it as well. So if I am hacking in TypeScript I can just goto definitions, and it will take me to the `@types`  files. Which is nice, but it's not super straight forward to exercise the API. That's easy with Joyride. When I'm hacking vscode with ClojureScript, wether with Joyride or not, I still have this ease of just checking how things work. But no access to the types. So I go a bit back and forth.